docs/reference/query-languages/query-dsl/query-dsl-nested-query.md

```diff
@@ -71,7 +71,7 @@ See [Multi-level nested queries](#multi-level-nested-query-ex) for an example.
 
 
 `score_mode`
-:   (Optional, string) Indicates how scores for matching child objects affect the root parent document’s [relevance score](/reference/query-languages/query-dsl/query-filter-context.md#relevance-scores). Valid values are:
+:   (Optional, string) Indicates how scores for matching child objects affect the root parent document’s [relevance score](/reference/query-languages/query-dsl/query-filter-context.md#relevance-scores). Default is `avg`, but nested `knn` queries only support `score_mode=max`. Valid values are:
```

Comment:
Mind also putting a reference on `query-dsl-knn-query.md` There is a section there on `Knn query inside a nested query`

Also, it seems like we need to also now update: https://github.com/elastic/docs-content/blob/main/solutions/search/vector/knn.md in a separate PR.